Src/Plugins/ValidatorBase.cs

```diff
@@ -56,6 +56,18 @@ public static string CreateReturnValueForCompromisedAsset(string asset, string u
                 nameof(ValidationState.Authorized) + $"#The '{user}' account is compromised for '{asset}'.";
         }
 
+        public static string CreateReturnValueForFileException(Exception e, string asset)
+        {
+            if (e.Message.Equals("The specified network password is not correct."))
+            {
+                return nameof(ValidationState.Unknown) + $"#The file '{asset}' is secure and we could not validate.";
+            }
+
+            return nameof(ValidationState.Unknown) +
```

Comment:
>ValidationState [](start = 26, length = 15)

This helper should only contain this code and the helper should be called CreateReturnValueForUnhandledException